features/ouders.feature

```diff
@@ -255,70 +243,10 @@ Rule: Wanneer alleen gegevens in groep 81, 82, 83, 84, 85 en/of 86 zijn opgenome
-
-  @proxy
-  Scenario: vondeling
-    Gegeven het systeem heeft een persoon met de volgende gegevens
-    | naam                | waarde    |
-    | burgerservicenummer | 999993008 |
-    En de persoon heeft een 'ouder' met de volgende gegevens
-    | naam                                            | waarde   |
-    | ouderAanduiding                                 | 1        |
-    | geslachtsaanduiding (04.10)                     | V        |
-    | datumIngangFamilierechtelijkeBetrekking (62.10) | 20190614 |
-    En de 'ouder' heeft alleen de volgende 'naam' gegevens
-    | naam                  | waarde |
-    | geslachtsnaam (02.40) | .      |
-    En de 'ouder' heeft GEEN 'geboorte' gegevens
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                          |
-    | type                | RaadpleegMetBurgerservicenummer |
-    | burgerservicenummer | 999993008                       |
-    | fields              | ouders                          |
-    Dan heeft de persoon met burgerservicenummer '999993008' een 'ouder' met alleen de volgende gegevens
-    | naam                 | waarde        |
-    | type                 | OnbekendOuder |
-    | indicatieOnbekend    | true          |
-    | ouderAanduiding      | 1             |
-
-  @proxy
-  Scenario: er is wel een identificerend gegeven geslachtsnaam van de ouder
-    Gegeven het systeem heeft een persoon met de volgende gegevens
-    | naam                | waarde    |
-    | burgerservicenummer | 999993008 |
-    En de persoon heeft een 'ouder' met de volgende gegevens
-    | naam                                            | waarde   |
-    | ouderAanduiding                                 | 1        |
-    | geslachtsaanduiding (04.10)                     | V        |
-    | datumIngangFamilierechtelijkeBetrekking (62.10) | 20190614 |
-    En de 'ouder' heeft alleen de volgende 'naam' gegevens
-    | naam                  | waarde            |
-    | geslachtsnaam (02.40) | Ibin binti Yalniz |
-    En de 'ouder' heeft GEEN 'geboorte' gegevens
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                             |
-    | type                | RaadpleegMetBurgerservicenummer    |
-    | burgerservicenummer | 999993008                          |
-    | fields              | ouders.ouderAanduiding,ouders.naam |
-    Dan heeft de persoon met burgerservicenummer '999993008' een 'ouder' met alleen de volgende gegevens
-    | naam                 | waarde |
-    | type                 | Ouder  |
-    | ouderAanduiding      | 1      |
-    En heeft de 'ouder' alleen de volgende 'naam' gegevens
-    | naam          | waarde            |
-    | geslachtsnaam | Ibin binti Yalniz |
+Rule: Wanneer van de ouder wel gegevens geregistreerd zijn, maar geen van de met fields gevraagde gegevens heeft een waarde, dan wordt er een 'ouders' zonder gegevens geleverd
```

Comment:
deze rule zegt hetzelfde als de sub-rule op regel 177

onderstaande scenario toont ook hetzelfde als het scenario op regel 231.

Dus een van de twee regels en een van de twee scenario's kan worden verwijderd

Reply:
De sub-rule op 177 is een @gba rule en die op 246 is een @proxy rule.  die moet misschien nog wel aangevuld worden zodat een onbekende ouder als lege ouder geleverd wordt door de proxy. 

Ook bij kinderen hebben we voor de proxy duidelijk gemaakt dat een onbekend kind of een vraag om lege kindgegevens terwijl er wel andere kindgegevens zijn een kind zonder gegevens oplevert. 

IHet is toch niet vanzelfsprekend dat een leeg kind/partner/ouder of een kind/partner/ouder  met alleen onbekend-waardes door de proxy als leeg kind/partner/ouder geleverd wordt ?